clowder-framework/catalog
# Feature/add name check to transformations catalog

- Added a check based on name and version before inserting a new transformation.
- Updated the name check to forward to an update page. That page still only returns a 501 error.

To test:
You may start up with an empty Mongo database.
Login and go to Contribute to upload an extractor.
If that extractor doesn't already exist in the database:
You should move on to the details page for that extractor
Else an extractor with that same name and version exists
Exact match only, so 2.0 and 2.0.0 will allow a new extractor to be uploaded
It should send you to a page stating that the transformation already exists
The URL should match: transformations/{transformation id}/update
There is no check on the other information, so it is irrelevant if some of the information changes or if it is all the same.
